src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -74,8 +74,12 @@ public static class MachineOptions {
         @Config.Comment({"Steam to EU multiplier for Steam Multiblocks.", "1.0 means 1L Steam -> 1 EU. 0.5 means 2L Steam -> 1 EU.", "Default: 0.5"})
         public double multiblockSteamToEU = 0.5;
 
-        @Config.Comment({"Whether machines should explode when overloaded with power.", "Default: true"})
-        public boolean doExplosions = true;
+        @Config.Comment({"Whether machines or boilers explosion damages terrain.",
+                "Note machines or boilers always explode when overloaded with power or met special conditions regardless this config.", "Default: true"})
```

Comment:
`machines and boilers` and also `or met with special conditions, regardess of....`